### ISSUE TITLE
Some settings get into effect only after running configure

### ISSUE THREAD

**Issue by zoharl3** [texstudio-org/texstudio#3245]
1. The background color of a selected text in the editor is dark blue. I open options> configure> syntax highlight. No matter what the background of the selected text is, when I close the config window, the bg color of a selected text is changed to the system default. I don't mind, but I wish it would have been that way when I open texstudio without going into configure.
2. Auto save (set to 1min) starts working only after going to option> config> adv. editor.

### Environment
- TeXstudio: 4.6.2
- Qt: 6.5.1
- OS: Win11
- TeX distribution: miktex

**Comment by sunderme** [comment]
2) works on linux
what is the entry in texstudio.ini: `Files\Autosave=...`

**Comment by sunderme** [comment]
then it should work. How do you determine that it does not ?

**Comment by sunderme** [comment]
what exactly is "background of the selected text " in the syntax highlighting gui ?

**Comment by zoharl3** [comment]
1. In option> config> syntax highlight, there's a search category, and one of the options is select. It's possible to change it fore/background. But even without touching anything, pressing okay sets the background color of a selected text to the OS's default.
2. When you change something in the editor, there's a disk icon that appears near on the tab of the current file, which indicates unsaved changes. Currently, however, the auto-save does seem to work. I'll try to keep an eye on it and see when it happens again.

**Comment by sunderme** [comment]
1) work as expected. It is the background used  when limiting the search scope to the current selection.

**Comment by zoharl3** [comment]
1. Indeed. So how do I set the background color of the current selection?

**Comment by sunderme** [comment]
this can only be changed by the window color set-up (https://texstudio-org.github.io/background.html#style-sheets)

**Comment by zoharl3** [comment]
I don't think you can just set a style sheet, and it requires handling an event.
But I don't really want to change the background color of the selection, and the default OS one is fine. I just wish it was like that when texstudio starts without the need to go into config to trigger some bug.

**Comment by sunderme** [comment]
I don't see any open issues here:
1) you can't set a different background for selections, this is designed behaviour.
2) autosave seems to work

**Comment by sunderme** [comment]
I have no idea what effect is described here as you can't change the background of selection ?
Please demonstrate with a screen cast.

**Comment by zoharl3** [comment]
Look at the selection color before after config:

https://drive.google.com/file/d/1cZ5tRxHQZWZl6-5tLiVY9w8vtTDO_UTV/view?pli=1

**Comment by sunderme** [comment]
1. it looks like the window style is changed.
2. it does not look like default win11. What exactly is changed.

**Comment by zoharl3** [comment]
It's high contrast mode. I don't think it matters since switching out of it (alt+shift+prtsc) doesn't affect the selection color.
